### PR TITLE
Feature/ge survey route component

### LINKED ISSUE #34: User story 1: Link to survey
As a user,
When I visit /crates, the first time 
I click on + subscribe 
I am taken to a style preferences page "/style-preferences."

Also refer to #7

### PR DESCRIPTION
**Pull request by GreysonElkins**
#### What's this PR do?
* Adds a test model for a survey object `web/src/common/surveys`
* Adds a route and empty component for `/style-preferences`
* Renders a component for surveys with questions and answers
#### Where should the reviewer start?
* Check out the file structure of the new directories
* Go to `localhost:3000/style-preferences`
#### How should this be manually tested?
Not ready for manual testing (or merge for that matter)
I haven't written tests for this yet for two reasons:
1. Once we design our actual survey, all of the values to test for would be changed and break any tests written now
1. There is no user functionality here yet
#### Any background context you want to provide?
* If you run into a missing dependency for storybook checkout issue #27 (you may just need to run delete `package-lock.json`, `node_modules`, and run `npm install`; this branch should have the package itself ready to go from `package.json`
* Originally was going to tackle issue #48 but determined that focusing on dynamic creation of surveys at this point was beyond the scope of the project.
* **Images cannot be imported** here in the same old react manner, and will crash the app. Instead we need to refer to the path of the image within the domain, like so: ``
* My next task is to turn this into a controlled form, which will likely involve moving the survey into state and changing a lot of what's here now. 
* Rebasing made my `drafted-PR` workflow messy (#50). It almost seemed like we might not be able to rebase a branch that's already been pushed to github? I may need to read a bit more on the workflow, but this is a rebased copy of the branch in the draft PR.
#### What are the relevant tickets?
close #34, #39 
#### Questions:
* Should we worry about the difference between React `Component` and `PureComponent` at this point? (@NakiNorton I meant to ask during class today if this is related to a `pure function`)
* Similarly, should we have a full understanding of the `withRouter(Component)` function in export?

### New Feature Submissions:
* [x] Have you added an explanation of what your changes do?
* [x] Have you linted your code locally prior to submission?
### Testing:
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?
* [ ] All new and existing tests passed?

### CODE REVIEW

File: code/web/src/modules/style/Survey.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+import { connect } from 'react-redux'
+import { Link, withRouter } from 'react-router-dom'
+import { Helmet } from 'react-helmet'
+
+// UI Imports
+import { Grid, GridCell } from '../../ui/grid'
+import Card from '../../ui/card/Card'
+import H3 from '../../ui/typography/H3'
+import H4 from '../../ui/typography/H4'
+import { black, white, grey } from '../../ui/common/colors'
```
Comment (NakiNorton) [inline code/web/src/modules/style/Survey.js:13]: Nice work utilizing the existing UI components for consistency.

**Review by NakiNorton — APPROVED** [review]
Great work creating the Survey component and getting the route setup! What do you think about renaming the `routes/style.js` file to 'routes/survey.js' (or styleSurvey)? I feel like using the word survey here will be great for developer empathy.

In response to your questions, I think these are both 'nice to knows'. Great to research if we have time but I did a quick google search and I don't believe they'll have any impact on the implementation of our feature.

**Comment by GreysonElkins** [comment]
@NakiNorton oh no you merged too quick! I was about to push the route name change 😂
let's go ahead and change it to `styleSurvey` and it'll get swapped in whoever merges next

**Comment by NakiNorton** [comment]
Oops! Okay sounds like a good plan!!

**Pull request merged**